aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -2152,6 +2155,25 @@ impl VMValidator for AptosVM {
             }
         }
 
+        if !self.features.is_zkid_enabled() || !self.features.is_open_id_signature_enabled() {
+            if let Ok(authenticators) = aptos_types::zkid::get_zkid_authenticators(&transaction) {
+                for (_, sig) in authenticators {
+                    if !self.features.is_zkid_enabled()
```

Comment:
nit: two ifs can be merged together?

Reply:
i personally find it easier to read when the cases are separated